### PR TITLE
Fix php errors and add a message to indicate price changes

### PR DESCRIPTION
**Pull request by safalliajoseph**
@shabananavas This MR fixes the php errors and also adds a note about the price change. could you please review. Thanks!

### CODE REVIEW

File: src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php [hunk 1]
```diff
@@ -192,6 +192,12 @@ public function formElement(
       '#attributes' => [
         'class' => ['option-widgets'],
       ],
+      '#prefix' => '<div class = "product__internal">'
```
Comment (shabananavas) [inline src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php]: Can you rename this class to "product__commerce_options__pricing_note" or something. Also, can you provide a screenshot of where it appears.

File: src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php [hunk 2]
```diff
@@ -192,6 +192,12 @@ public function formElement(
...
+      . $this->t('
+          IMPORTANT: Some options will have additional prices and
+          the product price will change depending on the option you select.'
```
Comment (shabananavas) [inline src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php]: > depending on the **options** you select

File: src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php (hunk 1 above)
Comment (safalliajoseph) [reply src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php]: I added this class as in accurate industries the yellow theming is done for this class. so its easier and we do not have to add styling for a new class :|

**Comment by safalliajoseph** [comment]
@shabananavas I have addressed your concerns, could you please review.

**Comment by shabananavas** [comment]
Looks good 👍

**Comment by safalliajoseph** [comment]
@thenchev https://www.drupal.org/project/3111714/issues/3142761#comment-13645419 - Could you please add in this

**Comment by krystalcode** [comment]
The code here seems to have already been merged.

**Pull request closed without merging**